src/vehiclelist_func.h

```diff
@@ -26,7 +26,7 @@ void FindVehiclesWithOrder(VehiclePredicate veh_pred, OrderPredicate ord_pred, V
 	for (const OrderList *orderlist : OrderList::Iterate()) {
 
 		/* We assume all vehicles sharing an order list match the condition. */
-		const Vehicle *v = orderlist->GetFirstSharedVehicle();
+		Vehicle *v = orderlist->GetFirstSharedVehicle();
```

Comment:
What's this change doing?

Reply:
Alowing non-const use of `v` in `veh_func`, which is a bit hidden by the templating.